go-controller/pkg/cni/types.go

```diff
@@ -45,6 +45,13 @@ type Request struct {
 	Config []byte `json:"config,omitempty"`
 }
 
+// CNIRequestMetrics info to report from CNI shim to CNI server
+type CNIRequestMetrics struct {
+	Command     command `json:"command"`
+	ElapsedTime float64 `json:"elapsedTime"`
+	HasErr      string  `json:"hasErr"`
```

Comment:
Seems like `HasErr` would make more sense as a `bool`, and the recipient can convert it to a string

Reply:
@danwinship will make that chagne.